CONTRIBUTING.md

```diff
@@ -352,3 +355,30 @@ To set this up, in the Before launch section of any Run/Debug configuration, do
 You only need to do most of these steps once, when you create the
 External Tool. Then you can use it in any run config that spawns the
 agent, to rebuild it first.
+
+## Debugging VS Code
+
+Sometimes, the Agent behaves differently when called from IntelliJ than
+when called from VS Code, and you may need to debug the same code paths
```

Comment:
What does in mean for the agent to be called from VSCode? When the extension calls VSCode APIs and hits agent shims?

Reply:
This is fair feedback; I'll rewrite it to use the right terminology.